js/modules/k6/experimental/tracing/module.go

```diff
@@ -0,0 +1,48 @@
+// Package tracing implements a k6 JS module for instrumenting k6 scripts with tracing context information.
+package tracing
+
+import (
+	"go.k6.io/k6/js/modules"
+
+	"github.com/dop251/goja"
+)
+
+type (
+	// RootModule is the global module instance that will create Client
+	// instances for each VU.
+	RootModule struct{}
+
+	// ModuleInstance represents an instance of the JS module.
+	ModuleInstance struct {
+		vu modules.VU
+	}
+)
+
+// Ensure the interfaces are implemented correctly
+var (
+	_ modules.Instance = &ModuleInstance{}
+	_ modules.Module   = &RootModule{}
+)
+
+// New returns a pointer to a new RootModule instance
+func New() *RootModule {
+	return &RootModule{}
+}
+
+// NewModuleInstance implements the modules.Module interface and returns
+// a new instance for each VU.
+func (*RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
+	// Any goja.Value exported to a Go struct will be mapped
+	// to the matching fields using the `js:""` tags.
+	vu.Runtime().SetFieldNameMapper(goja.TagFieldNameMapper("js", true))
+
```

Comment:
Is this just from that extension? As touching any of hte `Set*` method on the runtime from an extension is likely a big no-no. And in this particular case it is a breaking other APIs as k6 has a particular implementation of this.

Reply:
Good catch indeed! 👀 🙇🏻  Gonna look into removing this!

Reply:
Dropped it ✅